src/components/InitialCountForm.component.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+class InitialCountForm extends Component {
+  state = {countSubmitted: false, invalidInput: false}
+  displayCounter = ev => {

```

Comment:
good job on moving this logic to the parent component. you should delete it here though